src/app/utils/lifeEvents/event.js

```diff
@@ -0,0 +1,91 @@
+import Roll from 'roll';
+import { adventures } from '../secondary/adventures.js';
+import { boons } from '../secondary/boons.js';
+import { crime } from '../secondary/crime.js';
+import { arcaneMatters } from '../secondary/arcaneMatters.js';
+import { punishment } from '../secondary/punishment.js';
+import { supernaturalEvents } from '../secondary/supernaturalEvents.js';
+import { tragedies } from '../secondary/tragedies.js';
+import { war } from '../secondary/war.js';
+import { weirdStuff } from '../secondary/weirdStuff.js';
+
+export function event() {
+  const roll = new Roll();
+  const eventRoll = roll.roll('d100');
+  let adventureId,
+      boonId,
+      crimeId,
+      arcaneMatterId,
+      punishmentId,
+      supernaturalEventId,
+      tragedyId,
+      warId,
+      weirdStuffId;
+
+  if (eventRoll.result >= 1 && eventRoll.result <= 10) {
+    tragedyId = tragedies();
+    return { outcome: '0110', secondaryTable: 'Tragedies', secondaryTableResult: tragedyId, multiTable: false };
+  }
+
+  if (eventRoll.result >= 11 && eventRoll.result <= 20) {
+    boonId = boons();
+    return { outcome: '1120', secondaryTable: 'Boons', secondaryTableResult: boonId, multiTable: false };
+  }
+
+  if (eventRoll.result >= 21 && eventRoll.result <= 30) {
+    //TODO: '2130':	'You fell in love or got married. If you get this result more than once, you can choose to have a child instead. Work with your DM to determine the identity of your love interest.',
```

Comment:
Looks like you figured out how to handle these TODO comments in the implementation, they can go now so they don't end up in repo.